addons/calendar/models/calendar_event.py

```diff
@@ -556,6 +563,29 @@ def action_sendmail(self):
                 meeting.attendee_ids._send_mail_to_attendees('calendar.calendar_template_meeting_invitation')
         return True
 
+    def action_mass_mailing(self):
```

Comment:
action_mass_mailing indicates this will call mass mailing application ... would probably be better named like  "action_open_composer"